### PR TITLE
BITMAKER-2086: Integrate estela with Webpack

### PR DESCRIPTION
**Pull request by JoseArmandoChavezQuijahuaman**
# Description

Modify estela to use webpack and restructure the app.

# Issue

* [Issue ID](https://tasks.bitmaker.dev/issues/2086).

## Checklist before requesting a review

- [x] I have performed a self-review of my code.
- [x] My code follows the style guidelines of this project.
- [x] I have made corresponding changes to the [documentation](https://github.com/bitmakerla/estela/tree/main/docs).
- [x] New and existing tests pass locally with my changes.
- [x] If this change is a core feature, I have added thorough tests.
- [x] If this change affects or depends on the behavior of other _estela_ repositories, I have created pull requests with the relevant changes in the affected repositories. Please, refer to our [official documentation](https://estela.bitmaker.la/docs/).
- [x] I understand that my pull request may be closed if it becomes obvious or I did not perform all of the steps above.

### CODE REVIEW

**Comment by JoseArmandoChavezQuijahuaman** [comment]
@rodp63 I think it's ready to review.

File: estela-web/package.json [hunk 1]
```diff
@@ -3,39 +3,24 @@
...
   "scripts": {
-    "dev": "react-scripts start",
-    "build": "react-scripts build",
-    "test": "react-scripts test",
-    "eject": "react-scripts eject",
+    "build": "webpack --mode production",
+    "build:dev": "webpack --mode development",
+    "build:start": "cd dist && PORT=3333 npx serve",
+    "start": "webpack serve --open --mode development",
+    "start:live": "webpack serve --open --mode development --live-reload --hot",
```
Comment (rodp63) [inline estela-web/package.json:23]: We have a `build` script in production mode, do we need a `start` script with the production flag as well?

File: estela-web/src/custum.d.ts [hunk 2]
```diff
@@ -0,0 +1,4 @@
+declare module "*.svg" {
+    const content: any;
```
Comment (rodp63) [inline estela-web/src/custum.d.ts:2]: Please fix this GitHub warning

**Review by rodp63 — CHANGES_REQUESTED** [review]

File: estela-web/src/custum.d.ts (hunk 2 above)
Comment (JoseArmandoChavezQuijahuaman) [reply estela-web/src/custum.d.ts:3]: Fixed up

**Review by rodp63 — APPROVED** [review]
Looks good 👍

**Pull request merged**